File: core/src/main/java/com/pluxity/facility/floor/dto/FloorResponse.java
package com.pluxity.facility.floor.dto;

import com.pluxity.facility.floor.Floor;

public record FloorResponse(String name, String floorId) {
    public static FloorResponse from(Floor floor) {
        return new FloorResponse(floor.getName(), floor.getFloorId());
    }
}

File: common/src/main/java/com/pluxity/facility/floor/dto/FloorResponse.kt
package com.pluxity.facility.floor.dto

import com.pluxity.facility.floor.Floor

@JvmRecord
data class FloorResponse(val name: String?, val floorId: String?) {
    companion object {
        fun from(floor: Floor): FloorResponse {
            return FloorResponse(floor.getName(), floor.getFloorId())
        }
    }
}
